src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexFiltering.cpp

```diff
@@ -452,13 +452,18 @@ namespace OpenMS
     return OpenMS::Math::pearsonCorrelationCoefficient(pattern1.begin(), pattern1.end(), pattern2.begin(), pattern2.end());
   }
 
+
   double MultiplexFiltering::getAveragineSimilarity(const vector<double>& pattern, double m) const
+
   {
     // construct averagine distribution
     IsotopeDistribution distribution;
     vector<double> averagine_pattern;
     distribution.setMaxIsotope(pattern.size());
-    distribution.estimateFromPeptideWeight(m);
+    if (type_m_=="peptide") distribution.estimateFromPeptideWeight(m);
```

Comment:
style: space around comparison operators
use blocks of {} for multiple if else expressions